samples/mem_tg/mem_tg.h

```diff
@@ -234,6 +236,30 @@ class mem_tg : public test_afu {
     return handle_->get_token();
   }
 
+  void duplicate(mem_tg *duplicate_obj) const {
+    
+    duplicate_obj->count_       = this->count_;
+    duplicate_obj->loop_        = this->loop_;
+    duplicate_obj->wcnt_        = this->wcnt_;
+    duplicate_obj->rcnt_        = this->rcnt_;
+    duplicate_obj->bcnt_        = this->bcnt_;
+    duplicate_obj->stride_      = this->stride_;
+    duplicate_obj->pattern_     = this->pattern_;
+    duplicate_obj->mem_speed_   = this->mem_speed_;
+    duplicate_obj->name_        = this->name_;
+    duplicate_obj->afu_id_      = this->afu_id_;
+    //duplicate_obj->app_         = this->app_;
```

Comment:
Probably remove these commented-out bits and add a comment explaining that they are purposefully excluded.